kovri.md

```diff
@@ -9,12 +9,12 @@ Ring signatures obscure the sender.
 Stealth addresses prevent outside observers from knowing the destination of where Monero is sent to.
 And RingCT hides the amount of Monero that's transmitted.
 However, some personally identifiable information may be leaked at the network level when making a transaction.
-This is where Kovri comes in to fill this possible gap.
+This gap of privacy is closed by a technology called Kovri.
 
 ## 3) - The Term - Kovri
 
-Kovri is a free, decentralized application, which is based on the Invisible Internet Project's I2P open specifications.
-Kovri uses both encryption and innovate routing techniques to create a private overlay-network across the Internet.
+Kovri is a free, decentralized application, which is based on I2P.
```

Comment:
>which is based on I2P
>I2P open specifications

This needs to be very clear. It is not based on I2P, it is based on open I2P specifications. Your revision's language is less technical which is good for this document but we also need to be factually accurate.

We should keep the current official line of "Kovri is a free, decentralized, anonymity technology based on [I2P](https://getmonero.org/resources/moneropedia/i2p.html)'s open specifications".